### ISSUE TITLE
Update booster catalog to v48

### ISSUE THREAD

**Issue by knrc** [openshift-istio/openshift-ansible#25]
https://github.com/fabric8-launcher/launcher-booster-catalog/releases/tag/v48

**Comment by knrc** [comment]
On hold for now, Luke has to verify with the latest installer before we can update.

**Comment by knrc** [comment]
Switching over to maistra organisation, https://github.com/Maistra/openshift-ansible/issues/1
